src/app/actions/admin/utilsStripe.tsx

```diff
@@ -0,0 +1,77 @@
+"use server";
+
+import { discountCalculation } from "@/utils/uiUtils";
+import Stripe from "stripe";
+
+interface StripeProductInput {
+  data: {
+    title: string;
+    description: string;
+    price: number;
+    discountPercentage: number;
+  };
+}
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!);
+export async function addProductStripe(result: StripeProductInput) {
+  const product = await stripe.products.create({
+    name: result.data.title,
+    description: result.data.description,
+  });
+  const price = await stripe.prices.create({
+    currency: "usd",
```

Comment:
Create a shared constant for values like currency that you use for the entire application. This way, you'll always be sure that you use the same string (not 'usd' in one place and 'USD' in another) everywhere and you won't need to worry about it. Plus, it's much easier to update that value if needed.